rpcs3/Gui/SettingsDialog.cpp

```diff
@@ -503,7 +503,16 @@ SettingsDialog::SettingsDialog(wxWindow *parent, rpcs3::config_t* cfg)
 		cfg->core.hook_st_func = chbox_core_hook_stfunc->GetValue();
 		cfg->core.load_liblv2 = chbox_core_load_liblv2->GetValue();
 
-		cfg->rsx.renderer = cbox_gs_render->GetSelection();
+		// Translates renderer string to id for config.h
+		if (cbox_gs_render->GetString(cbox_gs_render->GetSelection()) == "Null")
```

Comment:
One small thing: could you just use the enum values instead of 0-1-2-3? Use rsx_renderer_type::DX12 or rsx::renderer_type::Vulkan instead for readability by future contributors.